### PR TITLE
Merge upstream

### PR DESCRIPTION
**Pull request by egegunes**

### CODE REVIEW

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| We run 0 out of 3|

commit: https://github.com/percona/percona-postgresql-operator/pull/309/commits/e51ab546abd1db8f4335412aad96fbd43e8145ab
image: `perconalab/percona-postgresql-operator:PR-309-e51ab546`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| We run 0 out of 3|

commit: https://github.com/percona/percona-postgresql-operator/pull/309/commits/66346f2282536a396cb8ff75be7d3f56396f7c17
image: `perconalab/percona-postgresql-operator:PR-309-66346f22`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator-v2/PR-309/675a6555/init-deploy.log) |
| demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator-v2/PR-309/675a6555/demand-backup.log) |
| start-from-backup | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator-v2/PR-309/675a6555/start-from-backup.log) |
| We run 3 out of 3|

commit: https://github.com/percona/percona-postgresql-operator/pull/309/commits/675a655587bccfc4ee112a422f785a2fb75283fb
image: `perconalab/percona-postgresql-operator:PR-309-675a6555`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator-v2/PR-309/675a6555/init-deploy.log) |
| demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator-v2/PR-309/675a6555/demand-backup.log) |
| start-from-backup | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator-v2/PR-309/675a6555/start-from-backup.log) |
| We run 3 out of 3|

commit: https://github.com/percona/percona-postgresql-operator/pull/309/commits/675a655587bccfc4ee112a422f785a2fb75283fb
image: `perconalab/percona-postgresql-operator:PR-309-675a6555`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator-v2/PR-309/57b9b74d/init-deploy.log) |
| demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator-v2/PR-309/57b9b74d/demand-backup.log) |
| start-from-backup | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator-v2/PR-309/57b9b74d/start-from-backup.log) |
| We run 3 out of 3|

commit: https://github.com/percona/percona-postgresql-operator/pull/309/commits/57b9b74d358a19e222019ed6c84c9f61301f8584
image: `perconalab/percona-postgresql-operator:PR-309-57b9b74d`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator-v2/PR-309/57b9b74d/init-deploy.log) |
| demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator-v2/PR-309/57b9b74d/demand-backup.log) |
| start-from-backup | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator-v2/PR-309/57b9b74d/start-from-backup.log) |
| We run 3 out of 3|

commit: https://github.com/percona/percona-postgresql-operator/pull/309/commits/57b9b74d358a19e222019ed6c84c9f61301f8584
image: `perconalab/percona-postgresql-operator:PR-309-57b9b74d`

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+---
```
Comment (hors) [inline .github/ISSUE_TEMPLATE/bug_report.md:1]: do we really need it? If you see some undeeded files please do not add it.

File: docs/config.toml [hunk 2]
```diff
@@ -0,0 +1,93 @@
+baseURL= ""
```
Comment (hors) [inline docs/config.toml:1]: I think we need to remove doc at all

**Review by hors — CHANGES_REQUESTED** [review]

File: docs/config.toml (hunk 2 above)
Comment (egegunes) [reply docs/config.toml]: https://github.com/percona/percona-postgresql-operator/pull/309/commits/120b2b9689a7b719975c93904ba71db5a2ffe42e

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (egegunes) [reply .github/ISSUE_TEMPLATE/bug_report.md]: https://github.com/percona/percona-postgresql-operator/pull/309/commits/120b2b9689a7b719975c93904ba71db5a2ffe42e

**Review by hors — APPROVED** [review]

**Pull request merged**